app/views/admin/editions/_conflict.html.erb

```diff
@@ -20,7 +20,7 @@
 <% if edition.can_be_associated_with_world_locations? %>
   <h2 class="govuk-heading-m">World locations</h2>
   <% if edition.world_locations.any? %>
-    <%= render "world_locations/list", world_locations: edition.world_locations %>
+    <%= render "admin/world_location_news/list", world_locations: edition.world_locations %>
```

Comment:
So does this mean that there's been options missing from the admin interface since this was renamed? 😱

Reply:
Yeah.  Although this only appears if you update a document at the same time as someone else and your update changes the world locations, so is very unlikely to have ever been an issue.